src/app/pages/fund/components/receive-stx-item.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Circle, color, Stack } from '@stacks/ui';
+
+import StacksIcon from '@assets/images/fund/stacks-icon.png';
+import QRCodeIcon from '@assets/images/fund/qr-code-icon.png';
+import { Caption } from '@app/components/typography';
+import { FundPageSelectors } from '@tests/page-objects/fund.selectors';
+
+interface ReceiveStxItemProps {
+  onReceiveStx(): void;
+}
+export const ReceiveStxItem = ({ onReceiveStx }: ReceiveStxItemProps) => {
+  return (
+    <Box
+      _focus={{
+        boxShadow: ['0px 8px 16px rgba(0, 0, 0, 0.08)', '0 0 0 3px #5546FF'],
+      }}
+      _hover={{
+        boxShadow: '0px 8px 16px rgba(0, 0, 0, 0.08)',
+        cursor: 'pointer',
+      }}
+      as="button"
+      border="1px solid"
+      borderColor={color('border')}
+      borderRadius="16px"
+      boxShadow="0px 1px 2px rgba(0, 0, 0, 0.04)"
+      data-testid={FundPageSelectors.BtnReceiveStx}
+      display="flex"
+      onClick={onReceiveStx}
+      textAlign="left"
+    >
+      <Stack alignItems="flex-start" p="loose" spacing="base">
+        <Stack alignItems="center" isInline spacing="tight">
+          <Box>
+            <img src={StacksIcon} width="40px" />
+          </Box>
+          <Stack isInline spacing="extra-tight">
+            <Circle bg="#C4C4C4" opacity="0.3" size="4px" />
+            <Circle bg="#C4C4C4" opacity="0.6" size="4px" />
+            <Circle bg="#C4C4C4" size="4px" />
+          </Stack>
```

Comment:
I'd make this its own component, took me a while to realise what it was/corresponded too. Or at least, a component wrapping all of the whole visual elements here 
![Uploading image.png…]()

Reply:
It looks like I can just export this as an svg ...so maybe that is better?